.pre-commit-config.yaml

```diff
@@ -20,7 +20,7 @@ repos:
   rev: v3.15.0
   hooks:
     - id: pyupgrade
-      args: ['--py37-plus']
+      args: ['--py38-plus']
```

Comment:
It will generate a impact on code, see below:

```
diff --git a/src/flask_jsonrpc/types.py b/src/flask_jsonrpc/types.py
index 01d09cd..23c3733 100644
--- a/src/flask_jsonrpc/types.py
+++ b/src/flask_jsonrpc/types.py
@@ -46,7 +46,7 @@ except ImportError:  # pragma: no cover
 
 # Python 3.8+
 try:
-    from typing_extensions import Final
+    from typing import Final
 except ImportError:  # pragma: no cover
     from typing import Final  # type: ignore  # pylint: disable=C0412
```

Because of that, I didn't this change, and for now it isn't a big deal.

Reply:
Moreover, run the `pre-commit` command and commit the changes,

```
$ pre-commit run --all-files
```

Let see if that change will not break the code, ;).

Reply:
I mean, to push the code to this branch for GitHub Actions run the tests around the Python 3.8 version.